### ISSUE TITLE
Encountering RejectCallerWithCode error when using Anvil

### ISSUE THREAD

**Issue by penandlim** [Destiner/ethcall#146]
Forked mainnet using anvil 0.1.0 (28312e7 2022-10-13T00:10:13.586314Z)

Turned on the logs by setting RUST_LOG env variable

`RUST_LOG=backend,api anvil -f $RPC_URL`

Then using latest ethcall and ethers.js, create a Provider() and call all() or tryAll()

Emits error like below
```
2022-10-13T02:20:01.419714Z TRACE backendhandler: received request basic address=0xca11bde05977b3631167028862be2a173976ca11
2022-10-13T02:20:01.419745Z TRACE backend: call return RejectCallerWithCode out: None gas 0 on block 15736119
```

This is probably due to overriding the from to the multicall address

https://github.com/Destiner/ethcall/blob/master/src/call.ts#L63

Since some EVM implementations have limitations like this, I think from should be changed to a non-contract address.
https://eips.ethereum.org/EIPS/eip-3607

**Comment by Destiner** [comment]
Hmm, interesting. In the future `from` will be configurable and will be `0x0` by default (#137). For now, I'm thinking how this may be fixed in a non-breaking way. Right now, probably using earlier version of ethcall is the best choice.

**Comment by Destiner** [comment]
Fixed in `5.0.1`
